webui/react/playwright.config.ts

```diff
@@ -6,6 +6,18 @@ import { defineConfig, devices } from '@playwright/test';
 import * as dotenv from 'dotenv';
 dotenv.config();
 
+const server_addess = process.env.PW_SERVER_ADDRESS
+let port: number;
+if (server_addess === undefined) {
+  port = 3001
+} else {
+  const match = server_addess.match(/\d+/)
+  if (match === null || isNaN(+match)) {
+    throw new Error(`Expected port number in PW_SERVER_ADDRESS ${server_addess}, ${match}`)
+  }
+  port = +match
```

Comment:
`match` return [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#return_value), so if i dont miss anything, `port = +match` wouldn't work as expected. Anyways i think `Number(server_addess)` is preferable

Reply:
i think + worked as a shortcut. I wonder what it does under the hood. Anyway, im just about to push a commit that fixes it